lib/src/bouncing_grid.dart

```diff
@@ -53,22 +53,14 @@ class LoadingBouncingGrid extends StatefulWidget {
     this.inverted = false,
     this.itemBuilder,
     this.duration = const Duration(milliseconds: 1500),
-  })  : assert(backgroundColor != null,
-            'loading_animations: property [backgroundColor] must not be null. Prefer using Colors.transparent instead.'),
```

Comment:
why did you remove the developer warnings?

Reply:
its non nullable now 
developer can't set null to it so condition won't be false at all

Reply:
lol right, that makes sense 🤣